### ISSUE TITLE
Unable to save a catalog 

### ISSUE THREAD

**Issue by yblatti** [easySDI/easySDI#865]
**Original author :** Blatti Yves

Unable to save a catalog:
```

Unknown column 'created' in 'field list' SQL=INSERT INTO `jos_sdi_catalog_resourcetype` (`ordering`,`state`,`catalog_id`,`resourcetype_id`,`created`,`created_by`,`guid`) VALUES ('2','1','2','2','2014-11-18 13:19:31','10000','ad3bcbbc-ec0e-ae04-c508-bcf16a899e3b') 
```

It seems to be a change introduced in: "7529":https://forge.easysdi.org/projects/easysdi/repository/revisions/7529/diff/branches/4.2.x/joomla/easysdi/com_easysdi_catalog/src/administrator/tables/catalogresourcetype.php

*Easysdi_catalogTablecatalogresourcetype* now extends *sdiTable* (previously : JTable) but database model doesn't fit (no _created_, _modified_, _..._by_ ... fields)

**Comment by yblatti** [comment]
**Original author :** Villemagne Jérôme

Hello Yves,

it was fixed in 4.2.x and thus should be fix in 4.3.x ; if you're agree, you can close it :-)

**Comment by yblatti** [comment]
**Original author :** Blatti Yves

Perfect, thanks
